core/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -468,7 +468,7 @@ public void testGreatest()
   {
     // Same types
     assertExpr("greatest(y, 0)", 2L);
-    assertExpr("greatest(34.0, z, 5.0, 767.0", 767.0);
+    assertExpr("greatest(34.0, z, 5.0, 767.0)", 767.0);
```

Comment:
hmm interesting that this used to pass 🤔

Reply:
ah it still parses into the expression just complains about syntax on stderr